src/Gulpfile.js

```diff
@@ -95,6 +120,13 @@ function buildCssPipeline(assetGroup, doRebuild) {
             throw "Input file '" + inputPath + "' is not of a valid type for output file '" + assetGroup.outputPath + "'.";
     });
     var doConcat = path.basename(assetGroup.outputFileName, ".css") !== "@";
+    if (!doRebuild) {
+        console.log("CSS will only rebuild if less files specified in 'inputs' are newer.");
+    }
+    else {
+        console.log("Force Rebuild is enabled, rebuilding all input files.");
+    }
```

Comment:
Don't understand why you've added this logging. Seems arbitrary - shouldn't be here.